### ISSUE TITLE
PhantomJS not working

### ISSUE THREAD

**Issue by davinderkumar** [zodern/meteor-up#682]
I am getting error when trying to 
$curl http://www.myaddresspin.com?_escaped_fragment_=
Internal Server Error

And, in mup logs, I see this:

> spiderable: phantomjs failed: Error: Command failed: /bin/bash -c exec phantomjs --load-images=no --ignore-ssl-errors=yes /tmp/meteor_2576971355

>  at ChildProcess.exithandler (child_process.js:200:12)
    at emitTwo (events.js:87:13)
    at ChildProcess.emit (events.js:172:7)
    at maybeClose (internal/child_process.js:862:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:222:5) 
stderr: 
Error: Meteor code must always run within a Fiber. Try wrapping callbacks that you pass to non-Meteor libraries with Meteor.bindEnvironment.
    at Object.Meteor._nodeCodeMustBeInFiber (packages/meteor.js:1075:11)
    at [object Object]._.extend.get (packages/meteor.js:1087:12)
    at [object Object].RouteController.lookupOption (packages/iron_router.js:143:29)
    at new Controller.extend.constructor (packages/iron_router.js:103:21)
    at [object Object].ctor (packages/iron_core.js:144:17)
    at Function.Router.createController (packages/iron_router.js:810:18)
    at Function.Router.dispatch (packages/iron_router.js:1335:25)
    at Object.router (packages/iron_router.js:624:12)
    at next (/bundle/bundle/programs/server/npm/node_modules/meteor/webapp/node_modules/connect/lib/proto.js:174:15)
    at Object.handle (packages/cfs_http-methods.js:444:12)

In local environment, phantomjs works without any error.

**Comment by davinderkumar** [comment]
Spiderable works fine when I deploy build with "debug: true" option in mup.js file under buildOptions section. Strange! Ref: https://www.npmjs.com/package/mup#build-options

**Comment by kenshinman** [comment]
Hi @davinderkumar , what fork of spiderable are you using.
I have same problem

**Comment by MichalW** [comment]
@davinderkumar @kenshinman 
You can use a different docker image with newest phantomJs version:
```
imageFrontendServer: 'vazco/meteord'
```

**Comment by richardwattie** [comment]
Just in case this helps anyone in the future...

I was experiencing this issue as well. I tried everything I could think of and everything all other forums suggested.

The fix for me was my server! I have a t2.nano with AWS EC2 and I had uploaded my apps with mup.

It took me a long time to realise this was the cause... but if I just have 2 apps running everything works ok. I then added a third app and this is when problems occurred. I'm guessing it is down to the basic server I was running not being able to handle 3 docker containers for my apps and a fourth for mongo.

I always had issues uploading my third app with mup as well. I'm guessing this might be part of the issue.

So the point is... your publications etc might all be ok but the time to get a response from the server itself rather than the app can have an affect on this too.

I've beefed up my server a notch and it started to work with 3 apps.
